posthog/helpers/dashboard_templates.py

```diff
@@ -107,41 +167,15 @@ def _create_default_app_items(dashboard: Dashboard) -> None:
             INSIGHT: "FUNNELS",
         },
         last_refresh=now(),
+        color="green",
+        order=5,
         is_sample=True,
         description="This is a sample of how a user funnel could look like. It represents the number of users that performed "
```

Comment:
The sample here after showing several graphs of usable information could be jarring/misleading. My thought process would be that the "sample" would make me rethink if the previous graphs were also sample data

Reply:
Ah yeah, that's a really good point. It's tough to think of a useful funnel that we can make on the default data...

I'm wondering if funnels shouldn't be here? Or we could just do a `Pageview -> Pageview -> Pageview` funnel?

@marcushyett-ph curious to get your thoughts on this one.

Reply:
I think we want to encourage people to dig into their data and start exploring. `Pageview -> Pageview -> Pageview` is a funny funnel, but great canvas to start exploring from. But only in "top to bottom" mode. Left to right is broken :(.

The knowledge that [only 45% of visitors do more than 2 pageviews](https://app.posthog.com/insights/9TB833eC) is already interesting enough to get me to start asking questions. Seeing the same [broken down by browser](https://app.posthog.com/insights/mZur6WXH) could be even interesting and get me asking even more questions.

Reply:
Agreed - just updated it to `Pageview -> Pageview -> Pageview` broken down by browser.

Super easy to iterate here if we have better ideas too.